### PR TITLE
Add `ixlib` param, options, and readme info.

### PR DESCRIPTION
**Pull request by paulstraw**

### CODE REVIEW

File: spec/ImgixTagSpec.js [hunk 1]
```diff
@@ -152,6 +153,19 @@ describe('ImgixTag', function() {
     });
   });
 
+  it('includes the `ixjsv` parameter when `imgix.config.includeLibraryParam` is `true`', function() {
```
Comment (jayeb) [inline spec/ImgixTagSpec.js]: Should we also have a test asserting that this config setting is `true` by default?

**Comment by jayeb** [comment]
The only thing I'd add here is making sure that this config variable can be set via a `<meta>` tag, like the others. It kinda seems like overkill, but if we're committed to the idea that you can configure and run this whole library through markup alone, I think we should drop it in.

File: spec/ImgixTagSpec.js (hunk 1 above)
Comment (paulstraw) [inline spec/ImgixTagSpec.js]: Yep. Adding tests for default config.

**Comment by paulstraw** [comment]
Good call. Thanks Jason!

**Pull request merged**